mappings/net/minecraft/recipe/InputSlotFiller.mapping

```diff
@@ -1,10 +1,10 @@
 CLASS net/minecraft/class_2955 net/minecraft/recipe/InputSlotFiller
-	FIELD field_13347 recipeFinder Lnet/minecraft/class_1662;
-	FIELD field_13348 craftingScreenHandler Lnet/minecraft/class_1729;
+	FIELD field_13347 matcher Lnet/minecraft/class_1662;
+	FIELD field_13348 handler Lnet/minecraft/class_1729;
```

Comment:
IMO `screenHandler` is better here.

Reply:
This is a field. `private final ScreenHandler screenHandler;` is not too redundant and is more clear for cases such as `this.screenHandler` versus `this.handler`.

Reply:
Yeah screen handler is really a weird case, like if you are a particle, you may have a particle manager and resource manager; these cases are confusing unfortunately. In general, `handler` is too vague.

Reply:
Except there's really only one handler. I don't see any possibility for confusion here.